server/routerlicious/packages/lambdas/src/scribe/summaryWriter.ts

```diff
@@ -55,6 +55,7 @@ export class SummaryWriter implements ISummaryWriter {
         private readonly summaryStorage: IGitManager,
         private readonly opStorage: ICollection<ISequencedOperationMessage>,
         private readonly enableWholeSummaryUpload: boolean,
+        private readonly lastSummaryMessages: ISequencedDocumentMessage[],
```

Comment:
nit: `lastSummaryLogtailMessages`?

Reply:
Since message is the parameter of the ILatestSummaryState interface, I use lastSummaryMessages to be consistent.
https://github.com/microsoft/FluidFramework/blob/main/server/routerlicious/packages/lambdas/src/scribe/interfaces.ts#:~:text=messages%3A%20ISequencedDocumentMessage%5B%5D%3B